ui/imports/utils/Utils.qml

```diff
@@ -961,6 +961,6 @@ QtObject {
     }
 
     function getUrlStatus(url) {
-        return globalUtilsInst.isValidURL(url)
+        return true
```

Comment:
`bool QUrl::isValid() const` just saying :)

Reply:
this validator checks if the HTTP `get` call returns 200. `QUrl::isValid` can't really replace it.

Reply:
ah, so it does a connectivity check... hmm

Reply:
We have that facility in DOtherSide too

Reply:
add TODO with link to GH issue created to make a decision if its needed or to make the call async

Reply:
I wonder how making the backend call async will help with calling it from a plain JS function

Reply:
I mean, you will never get any result this way

Reply:
A refactoring is needed. We'll need the async support in qml as well